applications/nfc/scenes/nfc_scene_start.c

```diff
@@ -28,7 +29,11 @@ void nfc_scene_start_on_enter(void* context) {
         nfc);
     submenu_add_item(submenu, "Saved", SubmenuIndexSaved, nfc_scene_start_submenu_callback, nfc);
     submenu_add_item(
-        submenu, "Add Manually", SubmenuIndexAddManualy, nfc_scene_start_submenu_callback, nfc);
+        submenu,
+        "Generate/Add Manually",
```

Comment:
I am not sure that Generate is necessary here. From common user's point of view these two are quite similar, I would leave just "Add Manually"

Reply:
"Add manually" seems to imply to me I might have to do a bit of work to get a file made, which isn't really the case when generating. How about "Create", or "Create new"?

Reply:
Talked with our UX/UI team, we came to the decision that we should leave just "Add manually"... May be we change this later, but for now let's keep just like it's now in dev
Please, make this change and we will merge you PR